### DISCUSSION TITLE
Regarding GSOC 2022.

### DISCUSSION THREAD

**Discussion by Siddhesh-Yadav** [react-native-elements/react-native-elements#3259] [General]
Is this organization going to participate in 2022 GSOC. If it is then, how can I contribute to this organization? needed help.

**Comment by hetanthakkar** [comment]
@Siddhesh-Yadav Most probably react native elements won't be participating in gsoc this year because of lack of maintainers. Personally i would recommend you to search some other organisation because this organization is somewhat biased when it comes to selection in gsoc. People with less number of contributions will get selected so i highly recommend to contribute elsewhere where your contributions matter.

**Reply by Siddhesh-Yadav** [reply]
sorry, but you are? a mentor or contributor or else. you don't have a member tag. Anyways thank you for your guidance.🙏🙏

**Reply by hetanthakkar** [reply]
I was a contributor

**Reply by Siddhesh-Yadav** [reply]
Are you gonna participate in gsoc 2022? did you participate last year?

**Reply by hetanthakkar** [reply]
I participated in gsoc 2021 from this organization, but couldn't make it due to bias. you can see my contributions in contribution section

**Comment by flyingcircle** [comment]
That is a long ways off. If someone else in the org wants to take that on, they can do it. But I don't think I'll have the time to be a mentor.

**Comment by hetanthakkar** [comment]
@flyingcircle Yeah! Considering the unfair selection done by other mentors its better not to be mentor again

**Comment by flyingcircle** [comment]
@hetanthakkar1  this is a final warning. Stop besmirching members of this org, or I will block you.

**Comment by arpitBhalla** [comment]
We aren't participating this year, but contributions are always welcome ;)

**Comment by theianmay** [comment]
Closing this discussion as it's outdated, but we will consider participating for the 2026 cohort. As always, any contributions are welcome in the meantime!

https://summerofcode.withgoogle.com/
